github-management/default-branch-migration.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Default Branch Migration
+linkTitle: Default Branch Migration
+description:  |
+  Instructions on how to migrate the default branch from master to main.
+weight: 99
+type: docs
+aliases: [ "/rename" ]
+---
```

Comment:
@mrbobbytables can you check if this makes sense? what should be the value of the `weight` field?